starter-code/src/Contacts.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from "react";
+import contacts from "./contacts.json";
+
+class Contacts extends Component {
+  state = {
+    firstFive: [
+      {
+        name: "Idris Elba",
+        pictureUrl:
+          "https://image.tmdb.org/t/p/w500/d9NkfCwczP0TjgrjpF94jF67SK8.jpg",
+        popularity: 11.622713,
+        id: "11731993-0604-4bee-80d5-67ad845d0a38"
+      },
+      {
+        name: "Jessica Chastain",
+        pictureUrl:
+          "https://image.tmdb.org/t/p/w500/nkFrkn5NZVGWb4b2X0yIcXezhyt.jpg",
+        popularity: 8.324357,
+        id: "17980511-75ca-48b0-bea8-462fec2ee43d"
+      },
+      {
+        name: "Johnny Depp",
+        pictureUrl:
+          "https://image.tmdb.org/t/p/w500/kbWValANhZI8rbWZXximXuMN4UN.jpg",
+        popularity: 15.656534,
+        id: "7dad00f7-3949-477d-a7e2-1467fd2cfc06"
+      },
+      {
+        name: "Emilia Clarke",
+        pictureUrl:
+          "https://image.tmdb.org/t/p/w500/j7d083zIMhwnKro3tQqDz2Fq1UD.jpg",
+        popularity: 16.211837,
+        id: "e14aa81d-b812-412d-bc4d-4a0d2c9c66f4"
+      },
+      {
+        name: "Leonardo DiCaprio",
+        pictureUrl:
+          "https://image.tmdb.org/t/p/w500/A85WIRIKVsD2DeUSc8wQ4fOKc4e.jpg",
+        popularity: 11.245333,
+        id: "b4d2c7b8-fdd5-426a-85bd-011c3f50a6c6"
+      }
+    ]
+  };
+
+  randomContact = () => {
+    let index = Math.floor(Math.random() * contacts.length);
+    let newContact = contacts[index];
+
+    this.state.firstFive.forEach(contact => {
+      if (contact.id === newContact.id) {
```

Comment:
Do I understand it correctly what you are trying to do here is that if you already have the contact in your contacts-list, then get a new random contact? 

If yes: Keep in mind that if the contact is already in your this.state.firstFive, then you change it, but only once! What you would have to do is to put this into a while loop that doesn't stop until you get a contact that is not found in your this.state.firstFive